requirements-dev.txt

```diff
@@ -1,3 +1,4 @@
 click>=7.0
 ruamel.yaml>=0.15.89
 flake8>=3.7.7
+
```

Comment:
Please remove the extraneous line.

As a tip: Please use `git add -up` and not just `git add` in combination with `git diff --staged` whenever you prepare your commits. In that way you are prompted to confirm and review every single chunk of change you introduce making it so that errors like this are much less likely to occur.

Reply:
Sure, thanks